### PR TITLE
Feat/send note

### PR DESCRIPTION
**Pull request by pedrobertao**
Adding Description/Note to SEND
"Description" is now "Data" at REQUEST/MAKE Payment

### CODE REVIEW

File: src/scenes/Send/index.js [hunk 1]
```diff
@@ -31,15 +32,21 @@ import getBalanceStore from '../../store/balance'
...
 class SendScene extends Component {
-  static navigationOptions = {
-    header: null
+  static navigationOptions = ({ navigation }) => {
+    return {
+      header: <NavigationHeader
+        title={tl.t('send.title')}
+        onBack={() => navigation.goBack()}
+      />
+    }
```
Comment (RenanCostaNascimento) [inline src/scenes/Send/index.js]: I thought header was supposed to be on the body.

File: src/scenes/Send/index.js [hunk 2]
```diff
@@ -148,6 +155,11 @@ class SendScene extends Component {
       return
     }
 
+    if (description.length > 500) {
+      this.setState({ error: 'Description too long' })
```
Comment (RenanCostaNascimento) [inline src/scenes/Send/index.js]: I18n.

File: src/scenes/SubmitTransaction/index.js [hunk 3]
```diff
@@ -183,16 +183,24 @@ class TransactionDetail extends Component {
...
-    const { contracts } = transactionData
+    const { contracts, data } = transactionData
 
     const contractsElements = buildTransactionDetails(contracts, tokenAmount)
     contractsElements.push(
       <DetailRow
-        key={'TIME'}
-        title={'TIME'}
+        key='TIME'
+        title='TIME'
```
Comment (RenanCostaNascimento) [inline src/scenes/SubmitTransaction/index.js]: I18n

File: src/utils/translations/br.js [hunk 4]
```diff
@@ -220,7 +220,8 @@ const send = {
   input: {
     token: 'MOEDA',
     to: 'PARA',
-    amount: 'QUANTIA'
+    amount: 'QUANTIA',
+    description: 'DESCRIÇÃO (OPCIONAL)'
```
Comment (RenanCostaNascimento) [inline src/utils/translations/br.js]: Update nl and fr as well.

File: src/scenes/Payments/Build.js [hunk 5]
```diff
@@ -137,7 +137,7 @@ class RequestPayment extends Component {
             />
             <Input
               innerRef={(input) => { this.description = input }}
-              label='DESCRIPTION (OPTIONAL)'
+              label={tl.t('send.input.description')}
```
Comment (RenanCostaNascimento) [inline src/scenes/Payments/Build.js]: I18n on line 152 (GENERATE REQUEST) --I can't comment there.

File: src/scenes/Payments/Scan.js [hunk 6]
```diff
@@ -41,71 +41,71 @@ class ScanPayment extends Component {
...
-    } catch (error) {
-      if (error.name === 'DataError') Alert.alert('Warning', error.message)
-      else Alert.alert('Warning', 'Payment code invalid. Please, scan a valid one')
-      this.setState({loading: false})
-      this.scanner.reactivate()
+        this.setState({loading: false})
+        navigation.navigate('MakePayScene', {payment: {address, amount, token, description}})
+      } catch (error) {
+        if (error.name === 'DataError') Alert.alert('Warning', error.message)
+        else Alert.alert('Warning', 'Payment code invalid. Please, scan a valid one')
```
Comment (RenanCostaNascimento) [inline src/scenes/Payments/Scan.js]: Translate the errors

File: src/scenes/Payments/Scan.js [hunk 7]
```diff
@@ -41,71 +41,71 @@ class ScanPayment extends Component {
...
-              background='transparent'
-              justify='center'
-              align='center'
-            >
+    render () {
+      const { navigation, loading } = this.props
+      return (
+        <Utils.Container>
+          <NavigationHeader
+            title='SCAN'
```
Comment (RenanCostaNascimento) [inline src/scenes/Payments/Scan.js]: I18n

File: src/scenes/Payments/Scan.js [hunk 8]
```diff
@@ -41,71 +41,71 @@ class ScanPayment extends Component {
...
-  }
+                </Utils.Text>
+              </Utils.View>
+            }
+            cameraStyle={{
+              height: '100%',
+              width: '100%',
+              justifyContent: 'flex-start'
+            }}
+            permissionDialogMessage='To scan the transaction the app needs your permission to access the camera.'
```
Comment (RenanCostaNascimento) [inline src/scenes/Payments/Scan.js]: I18n

File: src/scenes/Send/index.js (hunk 1 above)
Comment (carlospnav) [reply src/scenes/Send/index.js]: I think Juliano talked about that but no decision was made. I could be wrong, though.

Comment (dbuarque) [reply src/scenes/Send/index.js]: @julianorafael

Comment (pedrobertao) [reply src/scenes/Send/index.js]: Well... what is it ?

File: src/utils/translations/br.js (hunk 4 above)
Comment (pedrobertao) [reply src/utils/translations/br.js]: I'm not confident enough on doing that. Should we use google translator ?

**Pull request merged**